src/util/apiUrl.ts

```diff
@@ -0,0 +1 @@
+export const apiUrl = import.meta.env.VITE_API_URL
```

Comment:
- The variable names for the constants should be in upper snake case. eg: `API URL`
- Move the content of this file into `/src/constants/index.ts`